source/ed/build_helper.cpp

```diff
@@ -426,12 +452,25 @@ VJ builder::vj(const std::string& line_name,
                const std::string& uri,
                const std::string& meta_vj,
                const std::string& physical_mode){
-    return vj_with_network("base_network", line_name, validity_pattern, block_id,
+    return vj_with_network("base_network", line_name, "", validity_pattern, block_id,
+                           wheelchair_boarding, uri, meta_vj, physical_mode);
+}
+
+VJ builder::vj_with_route(const std::string& line_name,
```

Comment:
It's really a minor point and it's not mandatory for you to change this since it will change a lot of code, but for these builders we recently added some way to better define them based on linked setters.

this way you only define the parameters you want to change in the builder.

eg, you can replace 

```
b.vj_with_route("line:A", "route:A1", "111111", "", true, "vj:1")(some stoptimes...)
```

to 
```
b.vj("line:A").route("route:A1").uri("vj:1")(some stoptimes...)
```


to do this you only have to create a '_route' field en `ed::VJ` and add the 'route' setter in the class
defined as `VJ& route(const std::string& r) { _route = r; return *this; }` 

This way you don't have to provide lot's of default values